crossword-pdf-uploader/src/main/scala/com/gu/crossword/pdfuploader/CrosswordConfigRetriever.scala

```diff
@@ -43,7 +43,15 @@ trait S3CrosswordConfigRetriever extends CrosswordConfigRetriever {
     val configFileKey = s"crossword-pdf-uploader/$stage/config.properties"
     val configInputStream = s3Client.getObject("crossword-uploader-config", configFileKey).getObjectContent
     val configFile: Properties = new Properties()
-    Try(configFile.load(configInputStream)) orElse sys.error("Could not load config file from s3. This lambda will not run.")
```

Comment:
out of interest: what's the reason for this change?

Reply:
This follows a review from @andrew-nowak where he noted this was a bit non-idiomatic. 

If this fails, we just want to stop and not carry on and `orElse` returning something (but really exiting) is a bit surprising. `try {} catch {}` with the `sys.exit` is intended to be a bit clearer about what it's doing.

Reply:
got it, that makes perfect sense -- thanks!